docs/config-flatcar-v1_1-exp.md

```diff
@@ -0,0 +1,190 @@
+---
+title: Flatcar v1.1.0-experimental
+parent: Configuration specifications
+nav_order: 98
+---
+
+# Flatcar Specification v1.1.0-experimental
+
```

Comment:
Let's copy in the experimental config warning from the other experimental spec docs.